client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+
+import { Grid } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+
+import Mentions from "./Mentions";
+import Platforms from "./Platforms";
+import Navbar from "./Navbar";
+
+const dashboardStyle = theme => ({
+  rightGridContainer: {
+    borderLeft: '1px solid black',
+    height: '100%'
+  }
+});
+
+class Dashboard extends Component {
```

Comment:
Just wondering why Dashboard component uses React Component based while all other components are functional components. If it's for state management, I think useState would be just fine for handling this.

Reply:
I think I was just used to using react components

Reply:
I changed it to a functional component